MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+
+### DataFormTextEditorItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[KeyBoard](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTextItemBase.html#Syncfusion_XForms_DataForm_DataFormTextItemBase_KeyBoard)'| markdownify }}</td>
+<td>Keyboard</td>
+<td>Gets or sets the type of keyboard used to display in the text, multiline and password editor.</td>
+</tr>
+</table>
+
+### DataFormDateItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for time picker.</td>
+</tr>
+
+<tr>
+<td>{{'[MaximumDate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_MaximumDate)'| markdownify }}</td>
+<td>MaximumDisplayDate</td>
+<td>Gets or sets the maximun display date to restrict the visible dates of the date editor.</td>
+</tr>
+
+<tr>
+<td>{{'[MinimumDate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_MinimumDate)'| markdownify }}</td>
+<td>MinimumDisplayDate</td>
+<td>Gets or sets the minimum display date to restrict the visible dates of the date editor.</td>
+</tr>
+</table>
+
+### DataFormTimeItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTimeItem.html#Syncfusion_XForms_DataForm_DataFormTimeItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for Time Picker.</td>
+</tr>
+</table>
+
+### DataFormListItem
+
+The common properties of `Xamarin SfDataForm` list items (` DataFormPickerItem`, ` DataFormAutoCompleteItem`, ` DataFormDropDownItem` and ` DataFormRadioGroupItem` ) are added to [DataFormListItem] class in `.NET MAUI SfDataForm` 
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ItemsSource](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_ItemsSource)'| markdownify }}</td>
+<td>ItemsSource</td>
+<td>Gets or sets the collection which will get displayed in picker where the user can select from available items.</td>
+</tr>
+
+<tr>
+<td>{{'[DisplayMemberPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_DisplayMemberPath)'| markdownify }}</td>
+<td>DisplayMemberPath</td>
+<td>Gets or sets a path to a value on the source object that will get displayed in picker.</td>
```

Comment:
<td>Gets or sets a path to a value on the source object that will get displayed in the picker.</td>